src/Features/Core/Portable/Completion/SuggestionMode/SuggestionModeCompletionProvider.cs

```diff
@@ -16,7 +16,23 @@ internal abstract class SuggestionModeCompletionProvider : CompletionListProvide
 
         public override async Task ProduceCompletionListAsync(CompletionListContext context)
         {
-            var builder = await this.GetBuilderAsync(context.Document, context.Position, context.TriggerInfo, context.CancellationToken).ConfigureAwait(false);
+            var document = context.Document;
+            var position = context.Position;
+            var triggerInfo = context.TriggerInfo;
+            var options = context.Options;
+            var cancellationToken = context.CancellationToken;
+
+            CompletionItem builder;
+            if (options.GetOption(CompletionOptions.AlwaysShowBuilder))
+            {
+                var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+                builder = this.CreateEmptyBuilder(text, position);
```

Comment:
Yes, that seems a good idea. This is consistent with our existing behavior, but that's better behavior. :smile: I'll file a bug.